codify/deployment.go

```diff
@@ -74,8 +78,8 @@ func (k Deployment) Uninstall() string {
 	tpl := template.New(fmt.Sprintf("%s", time.Now().String()))
 	tpl.Parse(uninstall)
 	buf := &bytes.Buffer{}
-	k.i.Name = varName(k.i.Name)
-	err := tpl.Execute(buf, k.i)
+	k.Name = varName(k.Name)
```

Comment:
should we give varName() a better name while we are in here?

Reply:
gave it a longer, more descriptive name. sanitizeK8sObjectName